tests/unit/Http/Request/GetClientAddressCest.php

```diff
@@ -27,6 +30,35 @@ public function httpRequestGetClientAddress(UnitTester $I)
     {
         $I->wantToTest('Http\Request - getClientAddress()');
 
-        $I->skipTest('Need implementation');
+        $store = $_SERVER ?? [];
+
+        $this->setNewFactoryDefault();
+        $request = $this->container->get('request');
+
+        $_SERVER = [
+            'REMOTE_ADDR'          => '192.168.0.1',
```

Comment:
Can we add IPv6 as well?

Reply:
To be honest I have no idea about IPv6 and how they work. I just copied the tests to the correct file.

Reply:
Ok. Will add a task for 4.1 to do a full IPv6 check. It's getting more and more important these days.